Exiled.CustomItems/API/Features/Modifiers.cs

```diff
@@ -15,29 +17,15 @@ public struct Modifiers
         /// <summary>
         /// Initializes a new instance of the <see cref="Modifiers"/> struct.
         /// </summary>
-        /// <param name="barrelType"><inheritdoc cref="BarrelType"/></param>
-        /// <param name="sightType"><inheritdoc cref="SightType"/></param>
-        /// <param name="otherType"><inheritdoc cref="OtherType"/></param>
-        public Modifiers(int barrelType, int sightType, int otherType)
+        /// <param name="attachmentNames"><inheritdoc cref="Attachments"/></param>
+        public Modifiers(AttachmentNameTranslation[] attachmentNames)
```

Comment:
This is a breaking change.

Reply:
Breaking changes in relation to custom items and attachments are allowed in this pr, since they are not implemented currently anyway.

Reply:
Even if it's a breaking change, I'm gonna break it more by removing the struct and implementing an array of `AttachmentNameTranslation`.